src/mapml/control/AnnounceScale.js

```diff
@@ -0,0 +1,67 @@
+export var AnnounceScale = L.Control.extend({
+    options: {
+      position: 'bottomleft',
+      maxWidth: 100,
+      updateWhenIdle: true,
+    },
+  
+    onAdd: function (map) {
+      var container = L.DomUtil.create('div', 'accessible-scalebar');
+      let invisibleScale = L.control.scale(this.options);
+      invisibleScale.addTo(map);
+      container.appendChild(invisibleScale.getContainer());
+      let invisibleScaleLine = invisibleScale._container.getElementsByClassName('leaflet-control-scale-line')[0];
+
+      setTimeout(() => { 
+        container.setAttribute('aria-label', this._setText(invisibleScaleLine));
+        container.style.display = 'none';
+        
+      }, 0);
+  
+      map.on('zoomend moveend', () => {
+        container.style.display = '';
+        let invisibleScaleLine = invisibleScale._container.getElementsByClassName('leaflet-control-scale-line')[0];
+
+        container.setAttribute('aria-label',  this._setText(invisibleScaleLine));
+        container.style.display = 'none';
+      });
+
+      return container;
+    },
+
+    _pixelsToDistance: function (px, units) {
```

Comment:
It looks to me like the announced scale ratio calculates the length of the CSS line to be about 1/2 what it is physically. i.e. when it says "1 centimeter", it looks like about 2 centimeters to me.

Reply:
"96 dpi" is in inches so maybe that accounts for the discrepancy?

Reply:
hmm. The distances look pretty accurate on my screen. Do you know what zoom level it says 1cm?